src/javascripts/components/example.js

```diff
@@ -1,6 +1,4 @@
-// The iframe-resizer module seems to export it's methods in an odd way
-// So we use an underscore here.
-import _ from 'iframe-resizer'
+import iFrameResize from 'iframe-resizer/js/iframeResizer'
```

Comment:
Saves 15KB+ of JavaScript by not (accidentally) bundling the iframe content script too